src/main/java/com/bugsnag/Bugsnag.java

```diff
@@ -42,6 +42,44 @@ public void rejectedExecution(Runnable runnable, ThreadPoolExecutor executor) {
      * Initialize a Bugsnag client and automatically send uncaught exceptions.
      *
      * @param apiKey your Bugsnag API key from your Bugsnag dashboard
+     * @return The Bugsnag client instance
+     */
+    public static Bugsnag createBugsnag(String apiKey) {
+        return createBugsnag(apiKey, true);
```

Comment:
We should be consistent in how we name this across platforms - Android uses `init`, and others use `start`. @kattrali did you have any thoughts on this? The notifier library spec doesn't seem to be too clear on this point

Reply:
changed to `init`. Makes sense to be consistent with Android